controllers/storagecluster/cephconfig.go

```diff
@@ -66,6 +67,30 @@ func (obj *ocsCephConfig) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.S
 		Data: rookConfigOverrideData,
 	}
 	_, err := ctrl.CreateOrUpdate(context.Background(), r.Client, rookConfigOverrideCM, func() error {
+		// mon_target_pg_per_osd=200 added only during new CM creation, as setting it on existing clusters can cause data movement.
```

Comment:
To keep this simple, what if the new setting is only added to the `defaultRookConfig`? It's not critical for upgraded clusters, right?

Alternatively, instead of using the configmap at all, this setting could be the first one that we add to the CephCluster spec under the `CephConfig` setting. I'd suggest this for the preferred approach for all the ceph settings that don't need to be overridable by the user, and also are not settings that affect the mons at first cluster creation. This setting is just for the PG auto scaler, so it's fine to set later.